tests/unit/__init__.py

```diff
@@ -1,37 +1,58 @@
+import logging
 import os
 import unittest
 
 import boto3
 from moto import mock_iam, mock_s3, mock_sts
 
-from upload.common.upload_config import UploadConfig
+from upload.common.upload_config import UploadConfig, UploadDbConfig, UploadVersion
 
-os.environ['DEPLOYMENT_STAGE'] = 'test'
 os.environ['LOG_LEVEL'] = 'CRITICAL'
+# logging.basicConfig(level=logging.DEBUG)
```

Comment:
Can we remove these lines if their commented out?

Reply:
I'd rather leave them in.  It took me a not insignificant amount of time to figure out the exact incantation required to get the stupid Python logging system to actually emit log messages during tests.  This is valuable knowledge.  I can keep it in my notes, but it's useful for others too, so they don't have to waste the time I did.